### PR TITLE
getting started - gcp - add second method of creating service account using gcloud

### PR DESCRIPTION
**Pull request by ernestas2k**

### CODE REVIEW

File: docs/getting-started/configuring-gcp-credentials.md [hunk 1]
```diff
@@ -3,11 +3,33 @@
...
+- (recommended) Visit Google Cloud <https://console.cloud.google.com/> and make sure you have selected the right project. Open Cloud Shell at the top right side of menu bar.
+- (alternative) You can also use your local `gcloud` installation. Make sure you activated the right
+  project (`gcloud projects list` and `gcloud config set project {{desired-project-id-here}}`)
+
+### Generate service account
+
+Execute this online script (or use it as a reference to create service account manually)
+
+```bash
+wget https://raw.githubusercontent.com/castai/docs/main/docs/getting-started/configuring-gcp-credentials/script.sh && chmod +x ./script.sh && ./script.sh
```
Comment (viktorasm) [inline docs/getting-started/configuring-gcp-credentials.md:19]: maybe a shorter form woud work, e.g.
```
wget -O - http://path/to/script.sh | bash
```

File: docs/getting-started/configuring-gcp-credentials.md [hunk 2]
```diff
@@ -3,11 +3,33 @@
...
+
+### Prerequisites
+
+- (recommended) Visit Google Cloud <https://console.cloud.google.com/> and make sure you have selected the right project. Open Cloud Shell at the top right side of menu bar.
+- (alternative) You can also use your local `gcloud` installation. Make sure you activated the right
+  project (`gcloud projects list` and `gcloud config set project {{desired-project-id-here}}`)
+
+### Generate service account
+
+Execute this online script (or use it as a reference to create service account manually)
```
Comment (viktorasm) [inline docs/getting-started/configuring-gcp-credentials.md:16]: I would reword this a bit, to reassure user what will happen. Don't think we need the "reference" part, it's self-explanatory for people who can read shell scripts.

```suggestion
Run the script below. This will create a new service account with required permissions, enable required APIs and then print account access key.
```

File: docs/getting-started/configuring-gcp-credentials.md [hunk 3]
```diff
@@ -3,11 +3,33 @@
...
+
+```bash
+wget https://raw.githubusercontent.com/castai/docs/main/docs/getting-started/configuring-gcp-credentials/script.sh && chmod +x ./script.sh && ./script.sh
+```
+
+Select the displayed json in cloud shell using mouse (cloud shell will copy the selection automatically, don't use ctrl+c)
```
Comment (viktorasm) [inline docs/getting-started/configuring-gcp-credentials.md:22]: Maybe too much of hand holding, we're teaching someone with skills to run things in console, how to copy/paste text:). Maybe just something like  "Copy displayed json and paste into create gcp credentials screen", and that's enough?

File: docs/getting-started/configuring-gcp-credentials.md [hunk 4]
```diff
@@ -3,11 +3,33 @@
...
+  project (`gcloud projects list` and `gcloud config set project {{desired-project-id-here}}`)
+
+### Generate service account
+
+Execute this online script (or use it as a reference to create service account manually)
+
+```bash
+wget https://raw.githubusercontent.com/castai/docs/main/docs/getting-started/configuring-gcp-credentials/script.sh && chmod +x ./script.sh && ./script.sh
+```
+
```
Comment (viktorasm) [inline docs/getting-started/configuring-gcp-credentials.md:21]: In here I'd show what the expected result should look like, just to know what to look for, e.g. a truncated version of script execution output and then the json we're after (with example values).

File: docs/getting-started/configuring-gcp-credentials.md (hunk 1 above)
Comment (anjmao) [reply docs/getting-started/configuring-gcp-credentials.md:19]: Piping downloadable script into bash is considered as a bad practice. Some devs even consider this as "horrible" practice.

File: docs/getting-started/configuring-gcp-credentials.md (hunk 3 above)
Comment (ernestas2k) [reply docs/getting-started/configuring-gcp-credentials.md:22]: Let's do it, keep in mind ctrl+c won't copy stdout output properly in cloud shell, which results in corrupted json. I'll keep this note in the instructions.

**Review by viktorasm — APPROVED** [review]
